src/test/java/testresults/PostTestResultsNegMainLvlCancelled.java

```diff
@@ -701,7 +716,8 @@ public void testResultsIntegerVehicleType() {
         testResultsSteps.validatePostErrorData("vehicleType", "must be one of [psv, hgv, trl]");
     }
 
-
+    //TODO - possible problem with gateway
```

Comment:
These Ignores... is there a related issue for this?
e.g. should you be using the @Issue tag for this, to link the ignores to the problem?